docs/connector-development/tutorials/cdk-api-source/config-based/0-getting-started.md

```diff
@@ -0,0 +1,22 @@
+# Getting Started
+
+## Summary
+
+Throughout this tutorial, we'll walk you through the creation an Airbyte source to read data from an HTTP API.
+
+We'll build a connector reading data from the Exchange Rates API, but the steps we'll go through will apply to other HTTP APIs you might be interested in integrating with.
+
+## Exchange Rates API Setup
+
+Before we can get started, you'll need to generate an API access key for the Exchange Rates API.
+This can be done by signing up for the Free tier plan on [Exchange Rates API](https://exchangeratesapi.io/).
+
+## Requirements
+
+- Python >= 3.9
```

Comment:
we shouldn't need python if we ask the developers to run through the docker image